test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -24,7 +24,6 @@
 /*
  * @test
  * @bug 8015854
- * @requires (os.family == "mac")
  * @summary Tests HTML image as JButton text for unwanted padding on macOS Aqua LAF
  * @run main HtmlButtonImageTest
  */
```

Comment:
I believe you have run this test on CI systems on all platforms

Reply:
Thanks for catching that unedited summary. Fixed.